### DISCUSSION TITLE
separate "create channels" and "delete and modify channels" into separate permissions

### DISCUSSION THREAD

**Discussion by pekopeko-raa** [stoatchat/discussions#142] [Feature Suggestions]
in rivote web app or whatever, for ""server"" owner there are switches availible to regulate users' permissions:
![](https://a.pomf.cat/vhaeoe.png)
Revolt would be a great chat app if it had a way for users to create new channels themselves, but as things currently are, that permission is tied to the permission of having full power to delete any channel on a ""server"".

**Comment by Jackenmen** [comment]
I personally think it's important to separate "modify channels" from "delete channels" as the latter is a very destructive operation. There's a big difference between being able to modify channel name, description, icon (and nsfw switch?) and it's a different thing to be able to delete a channel and its whole message history with it.
I suppose there's not much that you can change about a channel right now so it might seem a little niche but if in future, stuff like slow-mode gets added, this would be even more good to have.
In Discord, this requires a usage of a bot to serve as a middle-man but I think this could be better solved with a separate permission.
